syntax = "proto3";

package journey11.kpubsubai;

message PBUniqueWorkRef {
  string _ref = 1;
}